Postman/Domain/DomainSurveysResources.postman_collection.json

```diff
@@ -0,0 +1,98 @@
+{
+	"info": {
+		"_postman_id": "9bca65f1-919b-4404-9869-166493a4db17",
+		"name": "DomainSurveysResources",
+		"description": "A controller for querying Survey Resources",
```

Comment:
Should we call it controller or endpoint?

Reply:
Out of scope: Is ca copy from Public Api help page, we should improve the descriptions in other story